### DISCUSSION TITLE
Markdown部分快捷键缺失

### DISCUSSION THREAD

**Discussion by makerjackie** [obsidianzh/forum#30] [Q&A常见问题]
英文论坛相关issue: https://forum.obsidian.md/t/set-hotkeys-for-creating-unordered-lists-ordered-lists-and-task-lists/4332/29

我的需求
* 多级标题快捷键
* 有序、无序列表快捷键（其他软件通常是 ctrl + 7/8）
* 选中几行，转换为代码块

不过目前这些还没有实现

**Comment by Quorafind** [comment]
可能需要有人写插件了，或者等官方适配，现在才支持到复制HTML内容的底部

**Reply by iris2jane** [reply]
win系统的话有ob helper，Q群群主开发的ob助手。
